source/main.cpp

```diff
@@ -313,6 +314,11 @@ int main(int argc, char *argv[])
 					else
 						SDL_SetWindowFullscreen(window, SDL_WINDOW_FULLSCREEN_DESKTOP);
 				}
+				else if(event.type == SDL_KEYDOWN
```

Comment:
I think we should also check if this is a new press, as if one has updated the keybinding, holding the new FF key will result in a continuous toggling of the FF state